packages/@lightningjs/ui-components/src/components/ListItem/ListItemSlider.d.ts

```diff
@@ -18,20 +18,49 @@
 
 import lng from '@lightningjs/core';
 import { TextBoxStyle } from '../TextBox';
-import type { StylePartial } from '../../types/lui';
-import ListItem, { ListItemStyle } from '.';
+import { StylePartial } from '../../types/lui';
+import ListItem, { ListItemStyle } from './ListItem';
 
-export type ListItemSliderStyle = ListItemStyle & {
+type ListItemSliderStyle = ListItemStyle & {
   paddingY: number;
   valueTextStyle: TextBoxStyle;
 };
-export default class ListItemSlider extends ListItem {
+
+declare namespace ListItemSlider {
+  export interface TemplateSpec extends ListItem.TemplateSpec {
+    /**
+     * object containing all properties supported in the [Slider component](?path=/docs/elements-slider--basic)
+     */
+    slider?: Record<string, unknown>;
+
+    /**
+     * current value of slider
+     */
+    value?: string;
```

Comment:
Shouldn't this be a number? (And changed below?)

Reply:
it absolutely should be